lib/ts/version.ts

```diff
@@ -1 +1 @@
-export const package_version = "0.0.55";
```

Comment:
I don't think this needs a version number change.

Reply:
Darko helped me understand that this will happen later and doesn't need to be part of the pr.

I made a new commit putting the version number back